extras/docker/Dockerfile

```diff
@@ -1,92 +1,21 @@
-# w3af.org
-# https://github.com/andresriancho/w3af/tree/master/extras
+FROM ubuntu:14.04
 
-FROM ubuntu:12.04
-MAINTAINER Andres Riancho <andres.riancho@gmail.com>
+ADD . /w3af
```

Comment:
I may be wrong, but I believe this way works best. It's not an original idea of mine I confess. I saw such organization and as you can see, smaller scripts are easier to maintain than a big script.